### ISSUE TITLE
Works for category, but broke password protection on pages

### ISSUE THREAD

**Issue by griffinacademy** [benchmarkstudios/wc-category-locker#6]
It worked perfectly for WooCommernce, but it broke the built in password protection on pages, is there any fix to this?

**Comment by lukasjuhas** [comment]
Hi @griffinacademy,

Could you test the password protection on pages while wc-category-locker is disabled? Just ot make sure that the plugin is really the cause of your issue. 

You should also try to disable all other plugins leaving just wc-category-locker enabled and see if the problem persists.

Let me know and we take it from there, thank you!

Lukas

**Comment by griffinacademy** [comment]
Yes I did that, and the standard pw protection works when disabled. 

I also tried pw protecting a page after it was enabled, not having a pw protect page before enabling with the same results, 
​​​​​​Thank you! 
December 29, 2017 3:14 PM, "Lukas Juhas" <notifications@github.com (mailto:%22Lukas%20Juhas%22%20<notifications@github.com>)> wrote:
	Hi @griffinacademy (https://github.com/griffinacademy), 

	Could you test the password protection on pages while wc-category-locker is disabled? Just ot make sure that the plugin is really the cause of your issue. 

	You should also try to disable all other plugins leaving just wc-category-locker enabled and see if the problem persists. 

	Let me know and we take it from there, thank you! 

	Lukas
